backend/game/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import include, path
+from rest_framework.routers import DefaultRouter
+from game.views import GameViewSet, PlayerViewSet
+
+# The router handles the creation of generic API urls such as `users/` or `users/<int:pk>/`,
+# as well as the root url
```

Comment:
Doc pas adaptée pour ce module. Je propose de garder dans nos modules qui utilisent le routeur la doc suivante : "The router handles the creation of generic API urls as well as the root url"